README.md

```diff
@@ -96,6 +96,15 @@ Supporting contacts this way has a number of downsides:
   * Feature detection is harder, and would likely need a new API such as `HTMLInputElement.isTypeSupported()` mimicking the [`MediaRecorder`](https://www.w3.org/TR/mediastream-recording/#dom-mediarecorder-istypesupported). Unlike existing specializations of file pickers, contacts would be unlikely to gracefully degrade to a general picker.
   * Extensibility is harder, as it would rely on additional parameters being passed to the mime type.
 
+#### Previous Standardization Attempts
+There have been multiple previous attempts to bring a Contacts API to the web:
+  * https://wiki.mozilla.org/WebAPI/ContactsAPI
+  * https://lists.w3.org/Archives/Public/public-device-apis/2009Apr/att-0001/contacts.html
+  * https://www.w3.org/TR/2010/WD-contacts-api-20100121/
+  * https://www.w3.org/TR/2015/NOTE-contacts-manager-api-20150602/
+
+This attempt differs in its approach to privacy, which was the main emphasis of the API when it was designed. Unlike previous attempts which allow for perpetual access after granted permission, or include a vague privacy model, this spec enforces UI restrictions which give users full control over shared data and limit abuse. For example, a picker model is enforced where the user always acts as an intermediary of the shared contact info with full control every time contacts are requested.
```

Comment:
For the three other proposals, it might be worth to briefly outline why they were halted, and why the current proposal changes that situation.

For the 2009 attempt, it looks like this was _donated_ by Nokia rather than formally put on the standardization track, and that nobody picked this up. We could mention this in the same way as Mozilla's proposal.

For the 2010 attempt, it is very similar indeed, and also includes the ability to save contact information—which is tracked as #16 in this proposal. It was shelved in 2014 in favour of Web Intents, which never materialized. Once this proposal matures from WICG incubation, the DAP likely continues to be a potential home for the work.

For the 2015 attempt, it builds upon the 2010 proposal adding more options for programmatic access to the user's address book, namely deleting individual and/or all contact information. This seems to be quite distantly separated from today's interest from implementations.

Reply:
Thanks, this was super useful